### PR TITLE
add repo update

### PR DESCRIPTION
**Pull request by cgroschupp**

### CODE REVIEW

**Comment by cgroschupp** [comment]
@ktrysmt Please also have a look here.

File: bitbucket.go [hunk 1]
```diff
@@ -358,12 +359,12 @@ func (e RepositoryEnvironmentTypeOption) String() string {
...
 
 type RepositoryEnvironmentOptions struct {
-	Owner           string `json:"owner"`
-	RepoSlug        string `json:"repo_slug"`
-	Uuid            string `json:"uuid"`
-	Name            string `json:"name"`
+	Owner           string                          `json:"owner"`
+	RepoSlug        string                          `json:"repo_slug"`
+	Uuid            string                          `json:"uuid"`
+	Name            string                          `json:"name"`
```
Comment (ktrysmt) [inline bitbucket.go:365]: Please use gofmt to fix indents.

**Review by ktrysmt — COMMENTED** [review]
Looks good, but.

File: bitbucket.go (hunk 1 above)
Comment (cgroschupp) [reply bitbucket.go:365]: This is the result from gofmt. Gofmt add spaces to align the Code.

Comment (ktrysmt) [reply bitbucket.go:365]: I’m glad to hear that. Thank you for your confirmation.

**Pull request merged**